src/pages/player-sdk-ios/getting-started.mdx

```diff
@@ -29,3 +31,5 @@ We recommend using the most recent version of Xcode.
 #### Minimum OS version
 
 The Player SDK requires iOS 8.0 or above.
+
+[contact us]: https://video.ibm.com/enterprise-video/contact
```

Comment:
Due to this line being added, we can alter line 23:
From this: `If you have any questions, please [contact us](https://video.ibm.com/enterprise-video/contact).`
to this: `If you have any questions, please [contact us].`

Because the contact us link is already defined.

Reply:
done